packages/catalog-model/src/entity/ref.ts

```diff
@@ -160,22 +160,27 @@ export function parseEntityRef(
  * @param ref The reference to serialize
  * @returns The same reference on either string or compound form
  */
-export function serializeEntityRef(ref: {
-  kind?: string;
-  namespace?: string;
-  name: string;
-}): EntityRef {
-  const { kind, namespace, name } = ref;
-  if (
-    kind?.includes(':') ||
-    kind?.includes('/') ||
-    namespace?.includes(':') ||
-    namespace?.includes('/') ||
-    name.includes(':') ||
-    name.includes('/')
-  ) {
-    return { kind, namespace, name };
+export function serializeEntityName(ref: EntityName | Entity): string {
```

Comment:
Thoughts on this @freben? Did it because I cba handling full names in the storage parts of this PR, but figured it might make sense to forbid `:/` in kind and namespaces, lettings us always serialize to a string